cloudinit/net/cmdline.py

```diff
@@ -101,7 +101,7 @@ def _klibc_to_config_entry(content, mac_addrs=None):
     provided here.  There is no good documentation on this unfortunately.
 
     DEVICE=<name> is expected/required and PROTO should indicate if
-    this is 'static' or 'dhcp' or 'dhcp6' (LP: #1621507).
+    this is 'none' (static) or 'dhcp' or 'dhcp6' (LP: #1621507).
```

Comment:
Can we include a reference to https://git.launchpad.net/~usd-import-team/ubuntu/+source/klibc/tree/usr/kinit/ipconfig/README.ipconfig#n32 which seems to document this?